deepmedic/frontEnd/configParsing/trainSessionParams.py

```diff
@@ -455,6 +468,9 @@ def get_path_to_load_model_from(self):
     def get_tensorboard_bool(self):
         return self.tensorboardLog
 
+    def get_norm_params(self):
```

Comment:
remove. Return the parameters together with the standard arguments, under the "Pre-processing" section